services/frontend/src/components/profileForms/formTitle/FormTitleView.scss

```
@@ -0,0 +1,17 @@
+.profileForm-title {
+  font-size: 1.2em !important;
+  margin: 0;
+}
+
+.profileForm-unsavedText {
+  font-size: 0.9em !important;
+  margin-left: 10px;
+  font-weight: normal;
+  font-style: italic;
+  opacity: 0.5;
+}
+
+.profileForm-extra {
+  display: inline;
+  float: right;
+}
```

Looks good, small comment I think we should slowly be moving to this form for our scss files to address #995

```suggestion
.profileForm {
  &-title {
    font-size: 1.2em !important;
    margin: 0;
  }

  &-unsavedText {
    font-size: 0.9em !important;
    margin-left: 10px;
    font-weight: normal;
    font-style: italic;
    opacity: 0.5;
  }

  &-extra {
    display: inline;
    float: right;
  }
}
```

---

Thank you for the reminder. Will do!